Source/Other/Extensions/DateFormatterExtension.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  DateFormatterExtension.swift
+//  Sequel Ace
+//
+//  Created by James on 3/11/2020.
+//  Copyright © 2020 Sequel-Ace. All rights reserved.
+//
+
+import Foundation
+
+extension DateFormatter {
+	
+	@objc public static var mediumStyleFormatter: DateFormatter = {
+			
+		let formatter = DateFormatter()
+		formatter.formatterBehavior = Behavior.behavior10_4
```

Comment:
I wonder if we really need this, or the default would be okay

Reply:
from https://web.archive.org/web/20100312011327/http://developer.apple.com/mac/library/documentation/Cocoa/Conceptual/DataFormatting/Articles/dfDateFormatting10_4.html

> For backwards binary compatibility, the default behavior for NSDateFormatter on Mac OS X version 10.4 is the v10.0 behavior. On Mac OS X version 10.5 and later, however, the default is the v10.4 behavior.

So I think we can leave it out.

Reply:
Nice :)

Reply:
Confirmed. Without setting behaviour, the formatter defaults to 1040, the same as setting it to `behavior10_4`

![Screenshot 2020-11-03 at 5 26 02 PM](https://user-images.githubusercontent.com/1576190/97968708-f9cdac00-1df9-11eb-85e8-ade9dbf809d5.png)